apps/store/themes/store/js/list_assets.js

```diff
@@ -377,3 +590,12 @@ $(window).load(function () {
         document.getElementById("categoryDropDown").title = document.getElementById("categoryDropDown").text.trim()
     }
 });
+
+$( document ).ready(function() {
```

Comment:
When we clicking on the taxonomy drop down item, it will send ajax request.So we can't read available asset in list_asset.js page. So when document ready time,  we are checking the asset availability and assign it to another variable.